### DISCUSSION TITLE
Waiting before send off on Bale Collector

### DISCUSSION THREAD

**Discussion by rento112** [Courseplay/courseplay#6934] [Q&A] [answered]
Is there a way to make it where the Bale Collector mode doesnt go deliver immediate after picking up all the current bales on the field, I want it to wait until it's full before going. I'm having it collect while the bales are being made.

**Comment by Tensuko** [comment, marked as answer]
I don't see much need for this.
While the bale collector is driving off with the first few bales + collecting the next one, the balder got enough time to make some new.
But you should also wait with starting the bale collector, untill there are some bales already made.

Right now, from what I know, the bale collector check for new bales when collected one, I don't see a reason, why we should wait.
What could be done is, if we reworked the waittime option, you can add a Waittime in your course, before he starts to collect new bales.

**Reply by rento112** [reply]
> 
> 
> I don't see much need for this.
> While the bale collector is driving off with the first few bales + collecting the next one, the balder got enough time to make some new.
> But you should also wait with starting the bale collector, untill there are some bales already made.
> 
> Right now, from what I know, the bale collector check for new bales when collected one, I don't see a reason, why we should wait.
> What could be done is, if we reworked the waittime option, you can add a Waittime in your course, before he starts to collect new bales.

The reasonn why is because I have my sale point pretty close to my farm. I've given up on using the normal barn for selling the bales. Gets too finnicky backing into them.

**Reply by Tensuko** [reply]
Place a Waitpoint before the stoppoint somwhere on the way back to the field and let the driver wait.
